lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -22,6 +48,17 @@
         Logger.log('edx.certificate.shared', data);
       });
     });
+
+    function shareOnFacebook(text) {
+        FB.ui({
+            method: 'feed',
+            name: text,
+            link: "${request.build_absolute_uri(reverse('cert_html_view', kwargs=dict(user_id=str(user.id),course_id=unicode(course_id))))}",
+            picture: "${request.build_absolute_uri(course_image_url)}",
+            caption: 'Caption like which appear as title of the dialog box', // TODO# need to update as per requirement
+            description: 'Small description of the post' // TODO# need to update as per requirement
+        });
```

Comment:
Thanks, @asadiqbal08! I think this is more a decision for Product and Dev to make. @pbaruah @mattdrayer If I understand Asad correctly, if we do not define something for "Description" it is automatically populated -- I'm not too clear on what it gets populated with, or what the likely scraped info is going to be. Do you think that it's OK to also leave the description field value undefined and have the automatic population happen?